articles/azure-functions/functions-reference-powershell.md

```diff
@@ -0,0 +1,424 @@
+        }
+    }
+}  
+```
+
+For more information, see [host.json reference](functions-host-json.md).
+
+## HTTP triggers and bindings
+
+HTTP and webhook triggers and HTTP output bindings use request and response objects to represent the HTTP messaging.  
+
+### Request object
+
+The Request object that's passed into the script comes from a type called `HttpRequestContext` which has the following properties:
+
+| Property  | Description                                                    |
+|-----------|----------------------------------------------------------------|
+| _Body_    | An object that contains the body of the request.               |
+| _Headers_ | A dictionary that contains the request headers.                |
+| _Method_  | The HTTP method of the request.                                |
+| _Params_  | An object that contains the routing parameters of the request. |
+| _Query_   | An object that contains the query parameters.                  |
+| _Url_     | The URL of the request.                                        |
+
+### Response object
+
+The Response object that you should send back comes from a type called `HttpResponseContext` which has the following properties:
+
+| Property      | Description                                                 |
+|---------------|-------------------------------------------------------------|
+| _Body_        | An object that contains the body of the response.           |
+| _ContentType_ | A short hand for setting the content type for the response. |
+| _Headers_     | An object that contains the response headers.               |
+| _StatusCode_  | The HTTP status code of the response.                       |
+
+### Accessing the request and response
+
+When you work with HTTP triggers, you can access the HTTP request the same way you would with any other input binding. It's in the `param` block. 
+
+In order to send back a response, you need to send back an `HttpResponseContext` object. Here's an example:
+
+`function.json`
+```json
+{
+  "bindings": [
+    {
+      "type": "httpTrigger",
+      "direction": "in",
+      "authLevel": "anonymous"
+    },
+    {
+      "type": "http",
+      "direction": "out"
+    }
+  ]
+}
+```
+
+`run.ps1`
+```powershell
+param($req, $TriggerMetadata)
+
+$name = $req.Query.Name
+
+Push-OutputBinding -Name res -Value ([HttpResponseContext]@{
+    StatusCode = $status
+    Body = "Hello $name!"
+})
+```
+
+The result of invoking this function would be:
+
+```
+PS > irm http://localhost:5001?Name=Functions
+Hello Functions!
+```
+
+## PowerShell version
+
+The following table shows the PowerShell version used by each major version of the Functions runtime:
+
+| Functions version | PowerShell version                             |
+|-------------------|------------------------------------------------|
+| 1.x               | Windows PowerShell 5.1 (locked by the runtime) |
+| 2.x               | PowerShell Core 6.1                            |
```

Comment:
I would suggest just calling this PowerShell Core 6 to avoid having to update the version
```suggestion
| 2.x               | PowerShell Core 6                              |
```